sdk/eventhub/eventhubs-checkpointstore-blob/samples/downlevelContainerClient.ts

```diff
@@ -0,0 +1,41 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to extend the Storage Blob Container Client
+  to downlevel the API version used when communicating with the service.
+
+  This may be useful if the environment you are targetting supports an
+  older version of Storage Blob than is officially supported by the
+  @azure/storage-blob SDK.
+*/
+
+import { ContainerClient, StoragePipelineOptions, RequestPolicyFactory } from "@azure/storage-blob";
+
+/**
+ * The DownlevelContainerClient overwrites the API version sent via the
+ * `x-ms-version` header to "2017-11-09".
+ */
+export class DownlevelContainerClient extends ContainerClient {
```

Comment:
Same comment as above on the name of the class. Also, consider adding the prefix "Storage" to the final name like "APISpecificStorageContainerClient"